### PR TITLE
8316106: Open source few swing JInternalFrame and JMenuBar tests

### PR DESCRIPTION
**Pull request by kumarabhi006**
Few closed JInternalFrame and JMenuBar swing tests are open sourced.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8316106](https://bugs.openjdk.org/browse/JDK-8316106): Open source few swing JInternalFrame and JMenuBar tests (**Bug** - P4)

### Reviewers
 * [Alexander Zuev](https://openjdk.org/census#kizune) (@azuev-java - **Reviewer**) ⚠️ Review applies to [9c6e3254](https://git.openjdk.org/jdk/pull/15735/files/9c6e3254bb47472d4622613ca080b6591a429c25)
 * [Tejesh R](https://openjdk.org/census#tr) (@TejeshR13 - Committer)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/15735/head:pull/15735` \
`$ git checkout pull/15735`

Update a local copy of the PR: \
`$ git checkout pull/15735` \
`$ git pull https://git.openjdk.org/jdk.git pull/15735/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 15735`

View PR using the GUI difftool: \
`$ git pr show -t 15735`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/15735.diff">https://git.openjdk.org/jdk/pull/15735.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/15735#issuecomment-1718973200)

### CODE REVIEW

**Review by azuev-java — APPROVED** [review]
LGTM

File: test/jdk/javax/swing/JInternalFrame/bug4268949.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            if1.setBounds(20, 20, 95, 95);
+            if2.setBounds(120, 20, 95, 95);
+            if3.setBounds(220, 20, 95, 95);
+            if1.setBackground(Color.red);
+            if2.setBackground(Color.blue);
+            if3.setBackground(Color.green);
+            if (!((if1.getContentPane().getBackground()).equals(Color.red))
+                || !((if2.getContentPane().getBackground()).equals(Color.blue))
+                || !((if3.getContentPane().getBackground()).equals(Color.green))) {
+                throw new RuntimeException("Test failed: JInternalFrame " +
```
Comment (TejeshR13) [inline test/jdk/javax/swing/JInternalFrame/bug4268949.java:52]: Moving out of SwingUtilities would avoid InvovationTarget Exception.

File: test/jdk/javax/swing/JInternalFrame/bug4309079.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    private static JDesktopPane desktop;
+    private static JInternalFrame f1;
+    private static JInternalFrame f2;
+    private static volatile boolean passed = true;
+    private static volatile Point p;
+
+    public static void main(String[] args) throws Exception {
+        try {
+            Robot robot = new Robot();
+            robot.setAutoDelay(100);
```
Comment (TejeshR13) [inline test/jdk/javax/swing/JInternalFrame/bug4309079.java:58]: Inconsistent w.r.t other tests (test/jdk/javax/swing/JInternalFrame/bug4732229.java)

File: test/jdk/javax/swing/JInternalFrame/bug4268949.java (hunk 1 above)
Comment (kumarabhi006) [reply test/jdk/javax/swing/JInternalFrame/bug4268949.java]: Updated.

**Review by TejeshR13 — APPROVED** [review]

**Pull request closed without merging**